src/draw/CesiumDraw.js

```diff
@@ -144,16 +184,24 @@ export class CesiumDraw extends EventTarget {
     this.entities_.forEach(entity => this.viewer_.entities.remove(entity));
   }
 
-  drawSketchPoint_(position) {
-    return this.viewer_.entities.add({
+  drawSketchPoint_(position, edit = false) {
```

Comment:
The function could be named better.
What about calling it `createSketchPoint` and adding it to the entity collection in the calling code?
Also, can you pass an option object instead of a boolean? `{edit: true}`.